src/services/authentication/index.js

```diff
@@ -1,7 +1,7 @@
 const { AuthenticationService, JWTStrategy } = require('@feathersjs/authentication');
 const { LocalStrategy } = require('@feathersjs/authentication-local');
 
-const { LdapStrategy, MoodleStrategy, IservStrategy } = require('./strategies');
+const { LdapStrategy, MoodleStrategy, IservStrategy, TSPStrategy } = require('./strategies');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break before this closing brace. (object-curly-newline)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4491705)